### DISCUSSION TITLE
Rails 7.2.0 is incompatible with mongoid >= 8.1.3

### DISCUSSION THREAD

**Discussion by mpalenciano** [mongodb/mongoid#5855] [General]
Hi there,

We currently have mongoid 9.0.1 and rails 7.1.4. When trying to upgrade rails to 7.2 we get:

`
Because rails >= 7.2.0, < 7.2.1 depends on activemodel = 7.2.0
  and rails >= 7.2.1 depends on activemodel = 7.2.1,
  rails >= 7.2.0 requires activemodel = 7.2.0 OR = 7.2.1.
And because mongoid >= 8.1.3 depends on activemodel >= 5.1, < 7.2, != 7.0.0,
  rails >= 7.2.0 is incompatible with mongoid >= 8.1.3.
So, because Gemfile depends on mongoid ~> 9.0
  and Gemfile depends on rails ~> 7.2,
  version solving has failed.
`

Thanks and regards

**Comment by alexbevi** [comment]
Hey @mpalenciano, this is something we're getting sorted out in #5852 presently and should have included in the next release of mongoid

**Reply by mpalenciano** [reply]
That is great! thanks a lot, and for the quick response too. Regards
